feat: add C implementation for `stats/base/dists/kumaraswamy/skewness` (#4542)
stdlib-js/stdlib: lib/node_modules/@stdlib/stats/base/dists/kumaraswamy/skewness/README.md

<!-- /.examples -->

<!-- C interface documentation. -->

* * *

<section class="c">

## C APIs

<!-- Section to include introductory text. Make sure to keep an empty line after the intro `section` element and another before the `/section` close. -->

<section class="intro">

</section>

<!-- /.intro -->

<!-- C usage documentation. -->

<section class="usage">

### Usage

```c
#include "stdlib/stats/base/dists/kumaraswamy/skewness.h"
```

#### stdlib_base_dists_kumaraswamy_skewness( a, b )

> Evaluates the skewness of a Kumaraswamy's double bounded distribution.

Reviewer: ```suggestion
Returns the [skewness][skewness] of a [Kumaraswamy's double bounded][kumaraswamy-distribution] distribution with first shape parameter `a` and second shape parameter `b`.
```

```c
double out = stdlib_base_dists_kumaraswamy_skewness( 1.0, 1.0 );
// returns ~0.0
```

The function accepts the following arguments:

-   **a**: `[in] double` first shape parameter.
-   **b**: `[in] double` second shape parameter.

```c

> double stdlib_base_dists_kumaraswamy_logpdf( const double x, const
> double a, const double b );

Reviewer: ```suggestion
double stdlib_base_dists_kumaraswamy_skewness( const
double a, const double b );
```

```

</section>

<!-- /.usage -->

<!-- C API usage notes. Make sure to keep an empty line after the `section` element and another before the `/section` close. -->

<section class="notes">

</section>

<!-- /.notes -->

<!-- C API usage examples. -->

<section class="examples">

### Examples

```c
#include "stdlib/stats/base/dists/kumaraswamy/skewness.h"
#include <stdlib.h>
#include <stdio.h>

static double random_uniform( const double min, const double max ) {
    double v = (double)rand() / ( (double)RAND_MAX + 1.0 );
    return min + ( v*(max-min) );
}

int main( void ) {
    double a;
    double b;
    double y;
    int i;
    for ( i = 0; i < 25; i++ ) {

>         a = random_uniform( 0, 10.0 );
>         b = random_uniform( 0, 10.0 );

Reviewer: ```suggestion
        a = random_uniform( 0.1, 10.0 );
        b = random_uniform( 0.1, 10.0 );
```
